src/Symfony/Component/Messenger/composer.json

```diff
@@ -18,10 +18,7 @@
     "require": {
         "php": ">=8.0.2",
         "psr/log": "~1.0",
-        "symfony/amqp-messenger": "^5.1|^6.0",
         "symfony/deprecation-contracts": "^2.1",
```

Comment:
can be removed too

Reply:
No. There are still deprecations. I'll work with them in a followup PR.